autopush/main.py

```diff
@@ -254,8 +238,11 @@ def connection_main(sysargs=None):
     # Internal HTTP notification router
     site = cyclone.web.Application([
         (r"/push/([^\/]+)", r),
-        (r"/notif/([^\/]+)", n),
-    ], default_host=settings.router_hostname)
+        (r"/notif/([^\/]+)", n)
+    ],
+        default_host=settings.router_hostname, debug=args.debug,
+        log_function=skip_request_logging
```

Comment:
It's mainly because cyclone wants to log every http request, which strikes me as dumb, and the only way to disable it was to null op the log function. If we want to log something, we can with twisted's log function.